.github/workflows/reusable-build.yml

```diff
@@ -13,6 +13,8 @@ on:
 env:
   REGISTRY: ghcr.io
   FULL_REF: ${{ inputs.tag && format('refs/tags/{0}', inputs.tag) || github.ref }}
+  # This must match the entry in rustup-toolchain.toml at the repository root
```

Comment:
Looks like no such file was committed, but it should be named `rust-toolchain.toml` I think, not `rustup-toolchain.toml`.

Reply:
Yeah, that was confusing. Renamed this to `rust-toolchain.toml` and committed a rustup-toolchain.toml (as per our offline discussion).